### ISSUE TITLE
Provide support for longdesc for complex images

### ISSUE THREAD

**Issue by mgifford** [ckeditor/ckeditor5#14263]
### ✔️ Expected result

I should be able to provide alt text or longdesc values for complex images:
- https://www.w3.org/WAI/tutorials/images/complex/
- https://www.w3.org/TR/html-longdesc/

### ❌ Actual result

I don't see this available. 

### ❓ Possible solution

There are examples above for consideration fork the W3C.

### Also see:
https://www.drupal.org/project/drupal/issues/3050928

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mgifford** [comment]
The HTML 5 attribute longdesc isn't supported:  
https://lists.w3.org/Archives/Public/public-html-admin/2014Aug/0028.html

There is a need to provide longer descriptions should still be supported and this can be done with aria-describedby.

https://cookiecrook.com/longdesc/details/  
https://www.w3.org/WAI/tutorials/images/complex/#example-2-description-containing-textual-information  
https://usability.yale.edu/web-accessibility/articles/links#aria

**Comment by mgifford** [comment]
[A11yfirst](https://a11yfirst.gitlab.io/custom/a11yfirst.html) already provides a good pattern for this
